Makefile

```diff
@@ -39,15 +39,15 @@ endif
 lint: fmt
 
 test: lint .develop
-	pytest ./tests ./propcache
+	pytest ./tests
 
 
 vtest: lint .develop
-	pytest ./tests ./propcache -v
+	pytest -v ./tests
 
 
 cov: lint .develop
-	pytest --cov propcache --cov-report html --cov-report term ./tests/ ./propcache/
+	pytest --cov src/propcache --cov-report html --cov-report term ./tests
```

Comment:
@bdraco with `.coveragerc` having all the necessary configuration, the `pytest` invocations should just have the `--cov` option without any arguments, which also makes sure that other parts of the layout are measured.

This probably also causes these weird paths under `/Users/` to show up, like https://app.codecov.io/gh/aio-libs/propcache/tree/master/Users%2Frunner%2Fwork%2Fpropcache%2Fpropcache%2Fsrc%2Fpropcache. And I see that `.py` files are displayed on the top level, while `.pyx` are under `src/` on Codecov.

`--cov` corresponds to the `source =` setting in Coveragepy which accepts both importable identifiers and file paths, which is ambiguous. To make it cleaner, `.coveragerc` marks the entire project as source and specifies `source_pkgs` which only works with importables.

Reply:
Oh.. there's actually another place that is missing — `.coveragerc`'s path mapping doesn't take into account `src` which is likely one of the main causes of broken path mapping. Drop in this section https://github.com/ansible/awx_plugins.interfaces/blob/fc44b24/.coveragerc#L5-L12 to fix it. It maps src to site-packages w/o using project-specific names, which makes it copy-paste-friendly too.